addons/website_slides_survey/controllers/main.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
```

Comment:
File could be named "website_profile.py" as it's tied to that.
(We try to reduce just using "main.py" all the time).